tests/core/src/cluster/entities.spec.ts

```
@@ -0,0 +1,60 @@
+import { Effect } from 'effect';
+import type { ExtensionEntity } from '@eventiva/core';
+import { Entity, CurrentAddress, CurrentRunnerAddress, make, type Request } from '@eventiva/core';
+
+describe('cluster/entities', () => {
+    describe('exports', () => {
+        it.effect('exports ExtensionEntity type', () =>
+            Effect.gen(function* () {
+                // Type should be available (compile-time check)
+                const _test: ExtensionEntity = undefined as any;
+                expect(_test).toBeDefined();
+            })
```

Biome (2.4.6)

[error] This generator function doesn't contain yield.

(lint/correctness/useYield)